integration_tests/test_mesh_lens_correction.py

```diff
@@ -41,6 +41,26 @@ def render():
     return render
 
 
+def test_coarse_mesh_failure(render, tmpdir_factory):
+    outdir = str(tmpdir_factory.mktemp("mesh_lens"))
+    out_html_dir = outdir
+    example['render'] = render_params
```

Comment:
We have been using something like
```python
example_for_input = dict(example, **{"render": render_params})
```
in our tests since this makes a copy of the example data and does not risk contaminating other tests in the case of changes via schema validation or assignments in the module.